config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# rubocop:disable Lint/PercentStringArray
+SecureHeaders::Configuration.default do |config|
+  config.x_frame_options = "DENY"
```

Comment:
Is this going to affect GA? I think that falls back to a frame for no-js

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options it stops your website rendering in an iframe

Reply:
We have

```
    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-N58Z5PG"
    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->
```

Reply:
thats us including an iframe not our website in an iframe on another website

Reply:
So I think it is these lines that will cause issues for the google iframe 

frame_ancestors: %w['self'],
frame_src: %w['self'],

So will do some investigation

Reply:
Ah gotcha. Looks good then

Reply:
Those are both report-only, so we can mop that up with the rest